libcxx/utils/libcxx/test/params.py

```diff
@@ -73,7 +73,7 @@
     "-Wno-self-move",
 ]
 
-_allStandards = ["c++03", "c++11", "c++14", "c++17", "c++20", "c++23", "c++26"]
+_allStandards = ["c++03", "c++11", "c++14", "c++17", "c++20", "c++23", "c++26", "c++2c"]
```

Comment:
What is the difference between 26 and 2c?

Reply:
Nothing, I thought I needed it, but I don't.